### PR TITLE
i18n(ja): Update Japanese `getting-started.md`

### PR DESCRIPTION
**Pull request by kyosuke**
#### What kind of changes does this PR include?

- Translated content

#### Description

- Updated Japanese `getting-started.md` to match the English update
    - https://github.com/withastro/docs/commit/2ec991dbf46a37f5fb181760dba8f2ac6d5504c2
    - https://github.com/withastro/docs/commit/1e417f16d1517da0fc649e6037b28d987765627d

### CODE REVIEW

File: src/pages/ja/getting-started.md [hunk 1]
```diff
@@ -2,78 +2,82 @@
 setup: |
     import Button from '../../components/Button.astro'
     import ContributorList from '../../components/ContributorList.astro'
+    import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 layout: ~/layouts/MainLayout.astro
 title: はじめに
 description: Astroの基本的な入門です。
```
Comment (myakura) [inline src/pages/ja/getting-started.md:8]:
```suggestion
description: Astroの基本的な紹介です。
```

File: src/pages/ja/getting-started.md [hunk 2]
```diff
@@ -2,78 +2,82 @@
...
+
+[インストールガイド](/ja/install/auto/)には、お気に入りのパッケージマネージャーでAstroをインストールするための完全な手順が記載されています。
 
 
 ## Astroを学ぶ
 
 Astroサイトの主要なコンセプトとパターンの例をご覧ください。
 
-📚 Astroの[ディレクトリ構成](/ja/core-concepts/project-structure/)についてもっと読む。
+📚 サイトに[最初のページを追加](/ja/core-concepts/astro-pages/)。
```
Comment (myakura) [inline src/pages/ja/getting-started.md:74]:
```suggestion
📚 サイトに[最初のページを追加する](/ja/core-concepts/astro-pages/)。
```

他が動詞で終わっているので合わせた方がいいかもなと思いました。

**Review by myakura — APPROVED** [review]
Looks good!

**Comment by kyosuke** [comment]
@myakura Thanks! I have applied all your suggestions.

**Pull request merged**